docs/about/partners.rst

```diff
@@ -9,15 +9,15 @@ ClinGen Somatic Working Group
 
 Personalized Oncogenomics at BC Cancer Agency
 ---------------------------------------------
 .. sidebar:: \ 
 
    .. image:: http://civicdb.org/assets/images/partners/bc_cancer_agency.png
 
-The BC Cancer Agency's `Personalized Onco-Genomics (POG) <https://www.personalizedoncogenomics.org/>`_ program is a clinical research initiative that is embedding genomic sequencing into real-time treatment planning for BC patients with incurable cancers. Cancer is a complex biological process. We categorize cancers according to their site of origin (e.g. lung, breast, liver, colon) as each one is different, but even within these groupings there are subtypes with differences in response to treatment and overall behaviour. The POG program is a collaborative research study including many BCCA oncologists, pathologists and other clinicians along with the Genome Sciences Centre (GSC), which aims to decode the genome – the entire DNA and RNA inside the cell – of each patient's cancer, to understand what is enabling it to grow. Using this genomic data in clinical decision-making should allow us to develop treatment strategies to block its growth, identify clinical trials that the patient may benefit from and potentially identify less toxic and more effective therapeutic options. The POG program, under the leadership of Dr Steven Jones and Dr Janessa Laskin, have partnered with CIViC to study the impact of including CIViC interpretations in our clinical decision making. 
+The BC Cancer Agency's `Personalized Onco-Genomics (POG) <https://www.personalizedoncogenomics.org/>`_ program is a clinical research initiative that is embedding genomic sequencing into real-time treatment planning for BC patients with incurable cancers. Cancer is a complex biological process. POG categorizes cancers according to their site of origin (e.g. lung, breast, liver, colon) as each one is different, but even within these groupings there are subtypes with differences in response to treatment and overall behaviour. The POG program is a collaborative research study including many BCCA oncologists, pathologists and other clinicians along with the Genome Sciences Centre (GSC), which aims to decode the genome – the entire DNA and RNA inside the cell – of each patient's cancer, to understand what is enabling it to grow. Using this genomic data in clinical decision-making should allow us to develop treatment strategies to block its growth, identify clinical trials that the patient may benefit from and potentially identify less toxic and more effective therapeutic options. The POG program, under the leadership of Dr Steven Jones and Dr Janessa Laskin, have partnered with CIViC to study the impact of including CIViC interpretations in our clinical decision making. 
```

Comment:
I think this paragraph should be reworded to be in third person (they instead of we) to differentiate CIViC from POG.

Reply:
This was indeed my intent. I see that I missed an "us".

Reply:
fixed